src/xdocs/checks/coding/index.xml

```diff
@@ -395,6 +395,16 @@
               Checks that overloaded methods are grouped together.
             </td>
           </tr>
+          <tr>
+            <td>
+              <a href="constructorsdeclarationgrouping.html#ConstructorsDeclarationGrouping">
+                ConstructorsDeclarationGrouping
+              </a>
+            </td>
+            <td>
+              Checks that overloaded constructors are grouped together.
```

Comment:
Remove `overliaded` it is not applicable for constructors.
Remove it completely from all your PR

Reply:
Done